src/main/java/seedu/address/logic/parser/FlashBackParser.java

```diff
@@ -6,20 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.address.logic.commands.AddCommand;
-import seedu.address.logic.commands.ClearCommand;
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.DeleteCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FilterCommand;
-import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.commands.ReviewCommand;
-import seedu.address.logic.commands.StatsCommand;
-import seedu.address.logic.commands.UndoCommand;
-import seedu.address.logic.commands.ViewCommand;
+import seedu.address.logic.commands.*;
```

Comment:
I think to follow the java coding standard, this should be changed back to listing imported class explicitly instead of the star import.

Reply:
Avoid using wildcard import, it would be better to list all the import explicitly

Reply:
I changed the checkstyle to allow wildcard imports since we import considerably from one package. Wildcard imports should be ok if you import more than 3 functions/constants from a package